docs/getting-started/genaichatbot.md

```diff
@@ -0,0 +1,206 @@
+
+### 2b. Web Vectorize Pipeline
+
+Continuing with the goal of ingesting and vectorizing our web content, here we have the Web Vectorize Pipeline. We want to assign each document a number sequence, or vector, to map the similarity and relationships between those documents. Here we selected OpenAI [ada-002](https://openai.com/blog/new-and-improved-embedding-model) model based on performance and cost. As some of the documents are very long, we split them into smaller chunks. Each chunk is sent to OpenAI’s ada model. Run the Pipeline using the “Play” button and data preview buttons appear between Gems. Open the data preview following the OpenAI Gem and see the schema now includes a vector (or “embedding”) provided by the OpenAI model for each text chunk.
+
+![Web Vectorize Pipeline](img/genai_web_vectorize.png)
+
+Once the document chunks have each been assigned a vector, these “embeddings” are stored in the Unity Catalog and to a [vector database](https://www.pinecone.io/learn/vector-database/) (eg Pinecone). We can use this Knowledge Warehouse to construct our LLM prompt in the next Pipeline.
+
+#### 2b.i Configuring the Web Vectorize Pipeline
+
+1. Verify the Vectorize OpenAI Gem is configured with Databricks scope `open_ai` and Databricks key `api_key`.
+2. Verify the vector_db Target Gem is configured with the Databricks scope `pinecone` and Databricks key `token`.
+
+## Step 3: Live Inference
+
+### 3a. Chatbot Live Pipeline
+
+Finally, we get to run the most exciting Pipeline! The Chatbot Live streaming Pipeline ingests messages from Slack and sends the question and the relevant context to OpenAI which provides an answer.
+
+After ingesting the Slack question message and doing some transformation steps, the Chatbot Live Pipeline queries OpenAI to create an embedding specifically for the question. Then, the Pinecone Lookup Gem identifies documents, based on their vectors, which could be relevant for the question. With the IDs and vectors in hand, we need to pull from the full document corpus the relevant documentation text. The Join Gem does exactly this: gets content for the relevant document IDs. Now we are well on our way to creating a wonderful prompt! The OpenAI Gem sends the relevant content chunks and the Slack question in a prompt to OpenAI, and the model returns an answer. Finally, the Pipeline writes the answer back to the Slack thread.
+
+![Chatbot Live Pipeline](img/genai_chatbot_live.png)
+
+#### 3a.1 Configuring the Chatbot Live Pipeline
+
+1. Verify the `slack_chat` Source Gem is configured with Databricks scope `slack` and Databricks Key `app_token`. While this token begins with `xapp-`, be sure not to use the plaintext value, as using the Databricks secret is a much more secure approach.
+2. Update the `only_user_msgs` Filter Gem with the Slack app member ID:
+   ![Slack App Member ID](img/genai_memberId.png)
```

Comment:
@slyons thanks for pointing out this image was missing! 
I added it. Did you notice another image is missing?

Reply:
That was the only one